packages/core/src/hydration/annotate.ts

```diff
@@ -595,6 +595,12 @@ function serializeLView(
       continue;
     }
 
+    // Serialize information about template.
+    if (tNode.tView) {
```

Comment:
I think we should do this conditionally, only when it's an LContainer (similar to what we had before this change):

```suggestion
    if (isLContainer(lView[i]) && tNode.tView) {
```

Would that retain the correct behavior or it was part of the issue?

I'm also wondering if we should place it after the "disconnected node" check (a few lines below)?

Reply:
Since we removed the serialization from the lContainer section below the disconnected part, I think this needs to be outside of the disconnected check to make sure we still serialize this info when its needed. But adding in the `isLContainer` check should be fine.

Reply:
This has been updated.